### PR TITLE
Trying to help people building grpc using Bazel under Windows.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: bazel/grpc_build_system.bzl [hunk 1]
```diff
@@ -30,6 +30,13 @@ load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
 # The set of pollers to test against if a test exercises polling
 POLLERS = ["epollex", "epoll1", "poll"]
 
+def if_windows(a):
+    return select({
+        "//:windows": [],
+        "//:windows_msvc": [],
+        "//conditions:default": a,
```
Comment (gnossen) [inline bazel/grpc_build_system.bzl]: Typo? This looks like an implementation for `if_not_windows`.

Comment (nicolasnoble) [reply bazel/grpc_build_system.bzl]: Ugh, yes, thanks.

Comment (nicolasnoble) [reply bazel/grpc_build_system.bzl]: Fixed.

**Comment by jtattermusch** [comment]
qq:
Is there a user report that describes the problem users are facing?
How come our own bazel build on windows is not affected? (we're able to build just fine AFAIK)

**Comment by nicolasnoble** [comment]
@jtattermusch yes, Tensorflow complained internally.

**Pull request closed without merging**